pkgs/tools/networking/reaver-wps-t6x/default.nix

```diff
@@ -13,9 +14,23 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libpcap sqlite pixiewps ];
 
-  prePatch = "cd src";
+  sourceRoot = "reaver-wps-fork-t6x-v${version}-src/src";
 
-  preInstall = "mkdir -p $out/bin";
+  configureFlags = "--sysconfdir=${confdir}";
+
+  installPhase = ''
+    mkdir -p $out/{bin,etc,share}
+    cp reaver.db $out/etc/
+
+    for prog in reaver wash; do
+      cp $prog $out/share/
+      cat > $out/bin/$prog <<EOF
+    [ -f ${confdir}/reaver/reaver.db ] || (mkdir -p ${confdir}/reaver; cp $out/etc/reaver.db ${confdir}/reaver/)
```

Comment:
```install``` cannot fix the case when the directory ```/var/db/reaver-wps-1.4/reaver``` exists but the file ```/var/db/reaver-wps-1.4/reaver/reaver.db``` does not (I just hit this problem).